### DISCUSSION TITLE
API very slow / returning 5xx errors

### DISCUSSION THREAD

**Discussion by wisnewskirobert** [perplexityai/api-discussion#53] [General]
hi there, 
I am testing the perplexity API to see if I can use it in a new app.  

It is taking around 47 seconds to return a response.  Out of my last 6 tests I got 3 5xx errors and 3 200s. 

1) is it normal to get that many 500 errors? 
2) is this latency normal?  

Not sure I will be able to use it with this kind of performance. Maybe its just a 1 off today? 

Here are the details of my call: 
`Company Information: {
  "id": "e0b0e4d3-1ffb-402e-906a-3d6540216208",
  "model": "llama-3.1-sonar-huge-128k-online",
  "created": 1728991890,
  "usage": {
    "prompt_tokens": 128,
    "completion_tokens": 741,
    "total_tokens": 869
  },
  "object": "chat.completion",
`

thanks,

**Comment by wisnewskirobert** [comment]
Also here are my parameters: 

max_tokens: 1000, // Adjust as needed
    temperature: 0.2,
    top_p: 0.9,
    return_citations: true,
    search_domain_filter: ["https://fr.indeed.com/", "https://www.glassdoor.fr/com/", "https://www.linkedin.com/"],
    return_images: false,
    return_related_questions: false,
    search_recency_filter: "month",
    top_k: 0,
    stream: false,
    presence_penalty: 0,
    frequency_penalty: 1

**Comment by aarashy** [comment]
Hey, I recommend using sonar-large instead; it is much more stable.
